index.bs

```diff
@@ -313,6 +313,35 @@ See also
 
 * [[DESIGN-PRINCIPLES#do-not-expose-use-of-assistive-tech]]
 
+<h3 class=question id=hidden-data>
+  Does data exposed by your specification carry related but distinct
+  information that may not be obvious to users?
+</h3>
+
+Features which enable users
+to share data with origins
+should ensure that such data
+does not carry related but distinct metadata
```

Comment:
Maybe whether it's related or not is not the point. "does not carry embedded, possibly hidden, information that the user is unaware is being relayed, and might not consent to being relayed if they were aware"?

Reply:
Tweaked the wording based on this feedback.